lib/parsers.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @fileoverview Defines a storage for parsers.
+ * @author Ives van Hoorne
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Public Interface
+//------------------------------------------------------------------------------
+
+class Parsers {
```

Comment:
I think having a separate module for this isn't necessary given that a regular `Map` basically does the same thing (aside from using `require`). Instead, I think this could just be included as part of the `defineParser` method. However, I don't really have a strong objection to doing it like this.

Reply:
Agreed. This feels like a pretty generic store - I think it would actually be clearer to use a `Map` as well.